tests/data/format/linewrap_summary/function_docstrings.py.out

```diff
@@ -1,13 +1,15 @@
 def func():
```

Comment:
This test update can only be done with the `two-pass` enabled. So this can't be extracted to another PR currently.